globus_automate_client/cli/callbacks.py

```diff
@@ -64,6 +64,47 @@ def json_validator_callback(body: str) -> str:
     return body
 
 
+def yaml_validator_callback(body: str) -> str:
+    """
+    A user supplied body can be a properly formatted YAML string or a the name
+    of a file that contains valid YAML.  This validator ensures the user
+    supplies a valid YAML or valid filename containing valid YAML. Returns the
+    valid JSON string.
+    """
+    # Callbacks are run regardless of whether an option was explicitly set.
+    # Handle the scenario where the default value for an option is empty
+    if not body:
+        return body
+
+    # Reading from a file was indicated by prepending the filename with the @
+    # symbol -- for backwards compatability check if the symbol is present
+    # remove it if present
+    body = body.lstrip("@")
```

Comment:
Does this and all the file handling still need to be here, or has the input callback insured that by the time this is called, file loading is complete? Sorry if I've lost sight of the overall flow within the diff.

Reply:
Both yaml_validator_callback and json_validator_callback are not used anywhere. Since we are handling it in a different way now, would it be preferable to remove them? I personally tend towards removing code that is no longer used but I understand there are different approaches to handling this.